src/test/java/uk/gov/justice/digital/delius/service/ConvictionServiceTest.java

```diff
@@ -186,6 +191,96 @@ public void addingACourtCaseCreatesAConvictionEventWithNewIndexMatchingNumberOfC
         verify(spgNotificationService).notifyNewCourtCaseCreated(any(Event.class));
     }
 
+    @Nested
+    class GetConvictionsWithRar {
+
+        @Test
+        public void convictionsReturnedIgnoresSoftDeletedEvents() {
+            when(eventRepository.findByOffenderIdAndActiveTrue(1L))
+                .thenReturn(asList(
+                    aEvent().toBuilder().eventId(99L).offenderId(1L).softDeleted(1L).build()
+                ));
+
+            assertThat(convictionService.convictionsWithActiveRequirementFor(1L, "F"))
+                .isEmpty();
```

Comment:
Is it worth pulling the code out of the RequirementTypeMainCategory entity?

Reply:
Done